README.md

```diff
@@ -0,0 +1,54 @@
+# PDF GLPI plugin
+
+[![License](https://img.shields.io/github/license/pluginsGLPI/pdf.svg?&label=License)](https://github.com/pluginsGLPI/pdf/blob/main/LICENSE)
+[![Project Status: Active](http://www.repostatus.org/badges/latest/active.svg)](http://www.repostatus.org/#active)
+[![GitHub release](https://img.shields.io/github/release/pluginsGLPI/pdf.svg)](https://github.com/pluginsGLPI/pdf/releases)
+[![GitHub build](https://travis-ci.org/pluginsGLPI/pdf.svg?)](https://travis-ci.org/pluginsGLPI/pdf/)
+
+![Screenshot](./docs/screenshots/pdf.gif)
+
+This plugin allow you to select and export informations of an equipment to PDF file.
+- equipment types from GLPI
+- equipment types from some plugins
+- additional data from some plugins
+- one or many object(s) in a file
+
+Configuration :
+- International currency
+- Text to be added at the end of the PDF
+
+![Screenshot](./docs/screenshots/config.png)
```

Comment:
```suggestion
```
I suggest that you don't put anything here about configuration (there are other options in the personal preferences). This should rather be in a documentation dedicated to the plugin. 

here https://github.com/pluginsGLPI/doc